packages/p2p/src/pages/my-ads/copy-advert-form/copy-advert-form.tsx

```diff
@@ -0,0 +1,252 @@
+            });
+        } else {
+            my_profile_store.payment_methods_list.map(({ text, value }) => {
+                if (payment_method_names.includes(text)) my_ads_store.payment_method_names.push(value);
+            });
+        }
+
+        return () => {
+            my_ads_store.payment_method_names = [];
+            my_ads_store.payment_method_ids = [];
+        };
+    }, []);
+
+    return (
+        <div className='copy-advert-form'>
+            <InlineMessage
+                message={
+                    <Localize i18n_default_text='Review your settings and create a new ad. Every ad must have unique limits and rates.' />
+                }
+                type='information'
+            />
+            <Formik
+                initialValues={{
+                    contact_info,
+                    default_advert_description: description,
+                    max_transaction: '',
+                    min_transaction: '',
+                    offer_amount: amount_display,
+                    order_completion_time: order_expiry_period > 3600 ? '3600' : order_expiry_period.toString(),
+                    payment_method_names,
+                    rate_type: rate_display,
+                    type,
+                }}
+                onSubmit={onSubmit}
+                validate={my_ads_store.validateCreateAdForm}
+                validateOnMount
+            >
+                {({ errors, handleChange, isSubmitting, isValid, touched }) => {
+                    return (
+                        <Form noValidate>
+                            <Text color='less-prominent' size='xxs'>
+                                <Localize i18n_default_text='Ad type' />
+                            </Text>
+                            <Text as='div' className='copy-advert-form__field' size='xs'>
+                                {type === buy_sell.BUY ? localize('Buy') : localize('Sell')}
+                            </Text>
+                            <Text color='less-prominent' size='xxs'>
+                                <Localize i18n_default_text='Total amount' />
+                            </Text>
+                            <Field name='offer_amount'>
+                                {({ field }: FieldProps) => (
+                                    <Input
+                                        {...field}
+                                        data-testid='offer_amount'
+                                        data-lpignore='true'
+                                        type='text'
+                                        error={touched.offer_amount && errors.offer_amount}
+                                        trailing_icon={
+                                            <Text
+                                                color={is_desktop ? 'less-prominent' : 'prominent'}
+                                                size={is_desktop ? 'xxs' : 's'}
```

Comment:
can we create 2 variables for these since the other Text components are using the same values